index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>TedX Nit Kurukshetra</title>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta name="description" content="TEDx NIT, Kurukshetra" />
+    <meta name="robots" content="index, follow" />
+    <link rel="author" href="http://gawds.in" />
+    <meta name="author" content="gawds" />
+    <!-- Open Graph data -->
+    <meta property="og:title" content="TEDx NITK" />
```

Comment:
Make the same changes to Twitter card and Schema.org markup for Google+ sections. It helps SEO to find our website easily.